egs/wsj/s5/steps/combine_ali_dirs.sh

```diff
@@ -61,6 +62,7 @@ done
 
 src_id=0
 temp_dir=$dest/temp
+rm -rf $temp_dir
```

Comment:
No -f flag on rm, please, better to do
[ -d $temp_dir ] && rm -r $temp_dir
I just don't like using the -f flag on 'rm' as a general policy.